spec/controllers/repositories_controller_spec.rb

```diff
@@ -100,4 +100,37 @@
         .to(change { Activity.count })
     end
   end
+
+  describe 'POST import_records' do
+    let(:repository) { create :repository, team: team }
+    let(:mappings) do
+      { '0': '-1', '1': '', '2': '', '3': '', '4': '', '5': '' }
+    end
+    let(:params) do
+      { id: repository.id,
+        team_id: team.id,
+        file_id: 'file_id',
+        mappings: mappings }
+    end
+    let(:action) { post :import_records, params: params, format: :json }
+
+    it 'calls create activity for importing inventory items' do
+      ImportRepository::ImportRecords.any_instance.stub(:import!)
```

Comment:
better syntax here should be someting like: 
`allow_any_instance_of(ImportRepository::ImportRecords).to receive(:import!).and_return(status: :ok)` 
Try with this 1...